CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## UNRELEASED
 
+BUG FIXES:
+* Control Plane
+  * Fix issue where after a `helm upgrade`, users would see `x509: certificate signed by unknown authority.`
+    errors when modifying config entry resources. [[GH-808](https://github.com/hashicorp/consul-k8s/issues/808)]
```

Comment:
I see what you did here, but curious if this should link to the actual PR instead of the issue?

Reply:
done!